### PR TITLE
complete dom-events

### PR DESCRIPTION
**Pull request by CuongNgMan**

### CODE REVIEW

File: dom-events/exercise-1.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+(() => {
+  const table = document.querySelector('#row-container')
+
+  const ROW = 100
+  const COL = 100
+
+  function createARow(col, start) {
+    const tr = document.createElement('tr')
+
+    let td = null
```
Comment (viktordavidovich) [inline dom-events/exercise-1.js]: We usually don't use `null` at the time we declare variable.
Just do 
` let td`

**Review by viktordavidovich — COMMENTED** [review]
I believe we're missing one part of the task 
`
Compare performance in adding listener to each cell click and 
event delegation.
`
Did you try to measure performance somehow in both cases?
(adding click listener to each cell and to the parent table)

File: dom-events/exercise-1.js (hunk 1 above)
Comment (CuongNgMan) [reply dom-events/exercise-1.js]: Already improved

**Comment by CuongNgMan** [comment]
A separate folder for exercise-01 is created with an explanation of the difference between event delegation and adding events to each child item. @viktordavidovich

**Review by viktordavidovich — APPROVED** [review]
Well done!

**Pull request merged**